net/magic-wormhole/Portfile

```diff
@@ -27,9 +27,9 @@ license             MIT
 maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
-checksums           rmd160  bbb9d49f4520e5168f95b35264a4fce7ce4732a7 \
-                    sha256  1b0fd8a334da978f3dd96b620fa9b9348cabedf26a87f74baac7a37052928160 \
-                    size    262269
+checksums           rmd160  0f01edf2ff481fa9f773b7470cd7e9f115952f0f \
+                    sha256  ac3bd68286270e7f149c06149a8e409e5fa34d7feb0e88844a26d29eed2d1516 \
+                    size    274564
 
 python.default_version  310
```

Comment:
This should now be using the current version of Python, which is 3.11 (`311`).

Reply:
`py-autobahn` and `py-humanize` are both missing python3.11 versions. Should I bump them as part of this PR or do them in separate PRs?

Reply:
You can add Python 3.11 subports for each of them as individual additional commits to this PR. (`py-humanize: add Python 3.11 subport`)

Reply:
If you want to upgrade their versions as well, that would be a separate PR ideally.

Reply:
I'll just add 311 versions, I don't want to risk breaking other stuff especially since our `py-humanize` is 0.5.1 and the [current version is 4.8.0](https://pypi.org/project/humanize/).